tools/distpartitioning/dist_lookup.py

```diff
@@ -0,0 +1,289 @@
+import numpy as np
+import os
+import pyarrow
+import torch
+
+from pyarrow import csv
+from gloo_wrapper import alltoallv_cpu
+
+class DistLookupService:
+
+    def __init__(self, input_dir, ntype_names, id_map, rank, world_size):
+        '''
```

Comment:
this docstring should go to the class.

Reply:
Added the docstring to the class appropriately.

Reply:
here is an example of how python docstring works.
https://github.com/dmlc/dgl/blob/master/python/dgl/graph_index.py#L22

Reply:
the docstring of the input argument should be part of the docstring of the class.

Reply:
Updated the docstring accordingly.